package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "next-with-apollo",
-  "version": "5.1.1",
+  "version": "6.0.0-beta.1",
```

Let's do 6.0.0 because @apollo/client v3 is already stable.

```suggestion
  "version": "6.0.0",
```